boards/renesas/rzg3s_smarc/rzg3s_smarc_r9a08g045s33gbg_cm33.dts

```diff
@@ -23,6 +24,34 @@
 		zephyr,shell-uart = &scif1;
 	};
 
+	aliases {
+		sw0 = &sw_1;
+		sw1 = &sw_2;
+		sw2 = &sw_3;
+	};
+
+	buttons {
+		compatible = "gpio-keys";
+
+		sw_1: button_1 {
+			gpios = <&gpio18 0 GPIO_ACTIVE_LOW>;
```

Comment:
either move GPIO driver commit before board and move the board changes in this commit into the board commit, or move the board changes here into a separate commit

Reply:
Thank you.
I moved this modification of GPIO to the board commit.